map/bookmark_manager.cpp

```diff
@@ -404,6 +404,59 @@ void BookmarkManager::ResetRecentlyDeletedBookmark()
   m_recentlyDeletedBookmark.reset();
 }
 
+size_t BookmarkManager::GetRecentlyDeletedCategoriesCount() const
```

Comment:
Is this method covered by any test? It should be used by UI to quickly retrieve the count without expensive parsing of all deleted categories.

Reply:
I fixed the current cpp test and added checks for the GetRecentlyDeletedCategoriesCount